examples/sample-pub/publisher.js

```diff
@@ -393,6 +396,17 @@ function startFlowAuto() {
     });
     return;
   }
+
+  if (flow === 'showContributionOptions') {
+    onEntitlements = (subscriptions) => {
```

Comment:
should we rename this `contributions`?

Reply:
`subscriptions` is the standard name of the parameter (corresponds to the API in `subscriptions.ts`).